openhands/core/config/openhands_config.py

```diff
@@ -109,6 +109,10 @@ class OpenHandsConfig(BaseModel):
     mcp_host: str = Field(default=f'localhost:{os.getenv("port", 3000)}')
     mcp: MCPConfig = Field(default_factory=MCPConfig)
     kubernetes: KubernetesConfig = Field(default_factory=KubernetesConfig)
+    vi_mode: bool = Field(
+        default=os.environ.get('OPENHANDS_VI_MODE', 'false').lower()
+        in ('true', '1', 't')
+    )
```

Comment:
Actually i think you can keep it default to False.

If you set `VI_MODE=true`, it will automatically be picked up by OpenHands configuration system and set this label to true, i just tested with this `VI_MODE=true uvx --python 3.12 --from git+https://github.com/ikuo/OpenHands@feat/cli-vi-mode openhands` and it seems to work pretty well!

Reply:
Another slight concern i have is putting `vi_mode` under `OpenHandsConfig` which could make it even more bloated :(

Maybe we could create a new class called "CLIConfig" to put configuration like this that's CLI specific?

Reply:
@xingyaoww Thank you for the feedback!
I've put `vi_mode` under `CLIConfig` and tweaked the code at [b4e9bd4](https://github.com/All-Hands-AI/OpenHands/pull/9287/commits/b4e9bd4bf03761a2100f81771e2f8612e606772a).
Additionally, to align with the style of use of environment variable, I'v renamed `OPENHANDS_VI_MODE` to `CLI_VI_MODE`.

I've confirmed that vi_mode defaults to `False` by testing with and without `CLI_VI_MODE` variable.

I hope this makes sense.